src/test/scala/org/hammerlab/guacamole/commands/jointcaller/ReadSubsequenceSuite.scala

```diff
@@ -23,12 +23,12 @@ class ReadSubsequenceSuite extends GuacFunSuite {
       TestUtil.makeRead("TCGAGCGA", "8M", 1), // snv
       TestUtil.makeRead("TNGAGCGA", "8M", 1) // contains N base
     )
-    val pileups = reads.map(read => Pileup(Seq(read), "chr1", 1, simpleReference.getContig("chr1")))
+    val pileups = reads.map(read => makePileup(Seq(read), "chr1", 1))
 
-    ReadSubsequence.ofFixedReferenceLength(pileups(0).head, 1).get.sequence should equal("C")
-    ReadSubsequence.ofFixedReferenceLength(pileups(0).head, 2).get.sequence should equal("CG")
-    ReadSubsequence.ofFixedReferenceLength(pileups(1).head, 6).get.sequence should equal("CGACCCTCG")
-    ReadSubsequence.ofFixedReferenceLength(pileups(3).head, 1).get.sequenceIsAllStandardBases should equal(false)
+    ReadSubsequence.ofFixedReferenceLength(pileups(0).elements.head, 1).get.sequence should equal("C")
```

Comment:
after a change above in the assembly caller, this was the last use of `Pileup.head`, the existence of which is a smell, so I unrolled these accesses.